composer.json

```diff
@@ -14,7 +14,6 @@
         }
     },
     "require-dev": {
-        "zendframework/zend-http": "^2.5",
```

Comment:
The tests require this https://github.com/snowio/magento2-idempotent-api/blob/master/Test/Unit/Model/DispatchPluginTest.php#L18 ? Zend changed its name to Laminas, there's probably a "Laminas" alternative we could use here

Reply:
Had a discussion with @convenient about this.
He said those tests were never in any CI, and they don't run